src/subscript/eclcompress/eclcompress.py

```diff
@@ -75,6 +76,11 @@ def eclcompress(
     totalsavings = 0
 
     for filename in files:
+
+        if file_is_binary(filename):
+            logger.info("Skipped %s, not text file", filename)
+            continue
```

Comment:
I think guessing what a file is based on its content is a bad solution, must be absolute last resort.

- What generates these files?
- Why are binary files ignored?
- Who wants these files compressed and for what purpose?

Reply:
I agree in general, but evaluate it to be worth it in this situation.

* The upside in disk storage savings is significant. It can be gigabytes pr. realization.
* Letting users inject only `FORWARD_MODEL ECLCOMPRESS` in the ert config makes it very simple for users, but requires binary sniffing.

It is already possible for users to be explicit in which files to compress, but I think the upside in actually getting compression done outweighs the downsides from implicitness, therefore the recommended configuration in documentation is do it the simple way.

Binary files and ASCII-files (that should be compressed) are both outputted by RMS, and the file format is a choice made by the geomodeller when setting up the file exports in RMS.

The better solution is have RMS do run-length encoding of ascii export by default, but that feature is not there (yet?)

Reply:
Thanks for explaining, and that makes total sense.